src/components/Simulator/screens/ConnectWeb3/Web3App.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Box,
+  type BoxProps,
+  Icon,
+  Text,
+  Flex,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import React from "react"
+import { GrMenu } from "react-icons/gr"
+import { EthGlyphIcon } from "../../icons"
+
+interface IProps extends BoxProps {}
+export const Web3App: React.FC<IProps> = ({ children, ...boxProps }) => (
+  <Box h="full" w="full" bg="background.highlight" {...boxProps}>
+    <Box p={1} bg={useColorModeValue("#e8e8e8", "#171717")}>
```

Comment:
This is a hook. Should be defined at the top of the function.

```tsx
export const Web3App: React.FC<IProps> = ({ children, ...boxProps }) => {
  const bgColor = useColorModeValue("#e8e8e8", "#171717")

  return(
  <Box h="full" w="full" bg="background.highlight" {...boxProps}>
    <Box p={1} bg={bgColor}>
```

Reply:
oh good catch, thanks